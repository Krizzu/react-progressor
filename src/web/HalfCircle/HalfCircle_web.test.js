import React from 'react';
import renderer from 'react-test-renderer';

import HalfCircleProgressor from './HalfCircle';


test('It renders without errors', () => {
  const tree = renderer.create(
    <HalfCircleProgressor />,
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
