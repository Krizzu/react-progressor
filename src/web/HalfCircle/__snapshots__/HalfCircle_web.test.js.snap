// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It renders without errors 1`] = `
<div
  style={
    Object {
      "display": "inline-block",
      "height": 120,
      "position": "relative",
      "width": 120,
    }
  }
>
  <div
    style={
      Object {
        "height": 60,
        "overflow": "hidden",
        "position": "relative",
        "width": 120,
      }
    }
  >
    <div
      style={
        Object {
          "borderBottomColor": "#8DE969",
          "borderLeftColor": "#eee",
          "borderRadius": "50%",
          "borderRightColor": "#8DE969",
          "borderStyle": "solid",
          "borderTopColor": "#eee",
          "borderWidth": 5,
          "boxSizing": "border-box",
          "height": 120,
          "left": 0,
          "position": "absolute",
          "top": 0,
          "transform": "rotate(45deg)",
          "transition": "transform 200ms linear",
          "width": 120,
        }
      }
    />
  </div>
</div>
`;
